src/components/App.js

```diff
@@ -1,19 +1,25 @@
 import React from 'react'
 import MapLayer from './MapLayer'
 import AppHeader from './AppHeader'
-import TimeLine from './TimeLine'
-import StatsSidebar from './StatsSidebar'
+import BottomNav from './BottomNav'
 import * as Actions from '../actions/appActions'
 import '../sass/App.sass'
 
 const App = () => {
   Actions.onLoad()
   return (
-    <div className='App'>
-      <AppHeader />
-      <StatsSidebar />
-      <MapLayer />
-      <TimeLine />
+    <div className='AppHolder'>
+      <div className='App'>
+        <div className='columns'>
+          <div className='column'>
+            <AppHeader />
+            <MapLayer />
+            <BottomNav />
+          </div>
+        </div>
+      <div className='columns'><div className='column is-12'><div className='gr-bottomApp'></div></div></div>
```

Comment:
Could we put these divs on separate lines to increase readability? Also, the innermost one should use the self-closing syntax, our linter complains about that.